app/views/static_pages/privacy_policy.html.erb

```diff
@@ -112,14 +112,14 @@
 
     <p class="govuk-body">
       Neither DfE nor its delivery partners will keep your personal data longer than we need it for the purpose(s) of
-      research, security and/or delivery of the service, and in any case not longer than XXXX years. Your data will be
-      deleted when DfE and its delivery partners no longer need it for these purposes, or when XXXX years have passed,
+      research, security and/or delivery of the service, and in any case not longer than 7 years. Your data will be
+      deleted when DfE and its delivery partners no longer need it for these purposes, or when 7 years have passed,
```

Comment:
Can we actually say 7 years? Does that mean people will be able to reclaim after 7 years?

Reply:
Checked and we can remove the reference to the dates.